src/formatDate.js

```diff
@@ -50,7 +50,55 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat[3];
+  const separatorTo = toFormat[3];
+  const arrayedDate = date.split(separatorFrom);
+  const store = [];
+  let day = 0;
+  let month = 0;
+  let year = 0;
+
+  for (let i = 0; i < toFormat.length; i++) {
+    if (toFormat[i] === 'DD') {
+      day = i;
+    };
+
+    if (toFormat[i] === 'MM') {
+      month = i;
+    }
+
+    if (toFormat[i] === 'YY' || toFormat[i] === 'YYYY') {
+      year = i;
+    }
```

Comment:
Use [`switch case`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) in cases where you constantly compare one value with others. Fix it everywhere